src/controllers/expenses.controller.js

```diff
@@ -0,0 +1,98 @@
+const expenseService = require('../services/expenses.service');
+const { getUserById } = require('../services/users.service');
+
+const getAllExpenses = (req, res) => {
+  const expenses = expenseService.getAllExpenses(req.query);
+
+  res.statusCode = 200;
+  res.send(expenses);
+};
+
+const getExpenseForPerson = (req, res) => {
+  const { id } = req.params;
+  const expense = expenseService.getExpenseById(id);
+
+  if (!expense) {
+    res.sendStatus(404);
+
+    return;
+  }
+
+  res.statusCode = 200;
+  res.send(expense);
```

Comment:
```suggestion
try {
  const expense = expenseService.getExpenseById(id);

  if (!expense) {
    res.sendStatus(404);

    return;
  }

  res.statusCode = 200;
  res.send(expense);
} catch (err) {
    res.statusCode = 500;
    res.send('Something went wrong);
}
```
Also, it would be better to wrap this bunch of code inside the try-catch block to handle errors related to the DB connection, etc. In this case, we should provide a client with a proper error code and avoid throwing an error object directly to the client.

Reply:
Also, try to implement these suggestions for all of your code